suborbital/atmo
# NATS based streams

This adds an additional NATS source type for streams. See the new `connections` section of the Directive, and the new `source` field on handlers in `example-project`

To test:
- Build `subo` from this PR: https://github.com/suborbital/subo/pull/87
- Start a NATS server using `docker run -p 4222:4222 -ti nats:latest`
- Build the example project using `subo build ./example-project`
- Run Atmo from this branch: `make atmo bundle=./example-project/runnables.wasm.zip`
- Run the Grav NATS tester (from the grav repo): `go run ./transport/nats/tester/main.go`

You should see the messages trigger the `helloworld-rs` runnable and see the response in the grav tester output
